### ISSUE TITLE
Closing terminal while connected to Sunfire remote server

### ISSUE THREAD

**Issue by RussellDash332** [nus-cs2030/2122-s1#18]
# Summary
This isn't required on setting up Java and Vim, but I managed to connect to Sunfire's remote server and wondering what happens if I exit the terminal.

## Description 
The last line of [the SSH Config Wiki](https://github.com/nus-cs2030/2122-s1/wiki/SSH-Config) states that I shouldn't do so and type ```exit``` instead. What happens if I accidentally close the terminal?

## Screenshots (if any): 
Just me connecting to Sunfire with an alias I've made in the ```.bashrc``` file.
![SSH stuff](https://user-images.githubusercontent.com/63991775/129411928-3b0a8e86-0dbf-4eec-9f1c-211d26cedfbd.png)

**Comment by Justinhoejj** [comment]
Just did some looking around. I think it is good practice to exit instead of closing the terminal to ensure that you are logged out of Sunfire safely and to avoid unintended access by others ( see 3.16 https://www.comp.nus.edu.sg/~cs1101x/3_ca/labs/lab0/unix_intro.html). 

There could also be processes that may not terminate if you close the terminal without exiting the server. 
https://unix.stackexchange.com/questions/67937/ssh-is-it-dangerous-to-close-terminal-without-exit.

In short, it is good practice to exit Sunfire before closing the terminal although nothing disastrous should happen from not doing so.

Hope Prof can verify this.

**Comment by RussellDash332** [comment]
Noted. Thanks for your help 😄 
Let's see what the Prof/TAs will say about this.

**Comment by hwkchia** [comment]
fyi, we shall be migrating to stu.comp.nus.edu.sg from now on.  We shall put up more instructions by the end of week 2.

**Comment by RussellDash332** [comment]
> fyi, we shall be migrating to stu.comp.nus.edu.sg from now on. We shall put up more instructions by the end of week 2.

Alright, thanks Prof! I'm closing this issue now.
